pipelines/templates/ci-common.yml

```diff
@@ -20,6 +20,7 @@ steps:
   - template: tasks/unitypackages.yml
     parameters:
       UnityVersion: ${{ parameters.UnityVersion }}
+  - template: tasks/upmpackages.yml

```

Comment:
Do we want to package for PR validation too? Could make it easier to download and test changes

Reply:
If it's super fast (<30 seconds) then that's fine (as part of the standard PR validation).

If not we could always spin up some other command (i.e. /azp run mrtk_upm) to generate those on demand

We've done a bunch of work to trim down pr validation time so looking to keep that as low as possible.

Reply:
A different project's CI that I added `npm pack`ing to takes ~8 seconds per package (and 1 second for copying and publishing all three of them at once), so it might be longer than 30 seconds for MRTK.

Reply:
yeah. its on the order of a handful of minutes